tests/test_gengogettext.py

```diff
@@ -6,8 +10,16 @@
 import gengogettext
 
 
-class TestGengoGettext(unittest.TestCase):
+@contextlib.contextmanager
+def ignoring(exception, errno=None):
```

Comment:
I'm guessing contextlib.supress is not good enough because it can't/doesn't check error nums?